pages/clustering/high-availability.mdx

```diff
@@ -243,6 +246,14 @@ health status of other instances and the time of last response since only the le
 SHOW INSTANCES;
 ```
 
+<Callout type="warning">
+
+When `SHOW INSTANCES` is being run, bolt server of the 1st coordinator (coordinator on which you run registration queries) will always 
+show IP address set to 0.0.0.0 instead of machine's IP address (container/pod when running in Docker/Kubernetes). All communication will
+still work but the output is wrong. We are working on improving this.
```

Comment:
s/but the output/but the displayed output

s/We are working/This is a known bug and we are working